tools/summaries/run_summary_generation.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/bin/bash
+
+if [ -z "$MIXER_API_KEY" ]
+then
+  echo "ERROR: Environment variable MIXER_API_KEY is not set."
+else
+  python3 -m venv .env
+  source .env/bin/activate
+  python3 -m pip install -r requirements.txt
+
+  # Get US States + Top 100 Cities summaries from saved Bard output
+  python3 build_summaries_from_csv.py priority-places-bard.tsv \
```

Comment:
Where did priority-places-bard.tsv come from? how might we re-generate it if needed?

Reply:
There's no script because it was manually generated. Added a section in `README.md` describing how it was generated.